### PR TITLE
Fixing 404 errors on Windows, MacOS, Linux & Portable Downloads (also fixes typo on MacOS description)

### PR DESCRIPTION
**Pull request by Essasetic**

### CODE REVIEW

**Comment by Essasetic** [comment]
Sorry, I am still a complete noob at GitHub and got confused with the two commits.

**Comment by joshuaboniface** [comment]
This is not a bug, the dirs just don't exist yet as I've yet to create them manually. I will do so now. The CI will automatically update them as new versions are created.

File: downloads/index.html [hunk 1]
```diff
@@ -208,32 +208,32 @@ <h4>Fedora/CentOS</h4>
...
-                        <a href="https://repo.jellyfin.org/releases/server/linux/stable" class="button button__accent">Stable</a>
-                        <a href="https://repo.jellyfin.org/releases/server/linux/nightly" class="button button__accent">Nightly</a>
+                        <a href="https://repo.jellyfin.org/releases/server/linux/versions" class="button button__accent">Stable</a>
+                        <a href="https://repo.jellyfin.org/releases/server/linux/versions/nightly" class="button button__accent">Nightly</a>
                         </p>
                     </div>
                     <div class="macos">
                         <h4>MacOS</h4>
-                        <p>Both installers (.exe) and manual ZIP archives (.zip) are provided.</p>
+                        <p>Both installers (.dmg) and manual ZIP archives (.zip) are provided.</p>
```
Comment (joshuaboniface) [inline downloads/index.html]: Though this is a valid fix and was a mistake on my part.

**Comment by joshuaboniface** [comment]
Just to record this: the intention is the `stable` and `nightly` just point to the latest one of each, which is really all someone should want to DL. Though I think adding a global link to the repository as a whole is a good idea. Let me add all that now in a separate PR (and I'll fix the `.dmg` typo).

**Pull request closed without merging**